<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateMenuView</class>
 <widget class="QWidget" name="TemplateMenuView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>800</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>450</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="UI_ViewResource.qrc">
     <normaloff>:/View/Icon/left_arrow.png</normaloff>:/View/Icon/left_arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>525</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>CONFIRM</string>
   </property>
  </widget>
  <widget class="QLabel" name="template_show_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>100</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>SELECT YOUR TEMPLATE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="template_menu_box">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>600</y>
     <width>440</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>TEMPLATE MENU</string>
   </property>
   <widget class="QStackedWidget" name="template_widget_stack">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>15</y>
      <width>430</width>
      <height>120</height>
     </rect>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="UI_ViewResource.qrc"/>
 </resources>
 <connections/>
</ui>
